src/plugins/unified_histogram/public/container/container.tsx

```diff
@@ -43,6 +44,7 @@ export type UnifiedHistogramContainerProps = {
   searchSessionId?: UnifiedHistogramRequestContext['searchSessionId'];
   requestAdapter?: UnifiedHistogramRequestContext['adapter'];
   isChartLoading?: boolean;
+  table?: Datatable;
```

Comment:
I expected to see info about a request for the grid data but it shows that data was requested for a visualization:

<img width="500" alt="Screenshot 2024-01-23 at 11 03 51" src="https://github.com/elastic/kibana/assets/1415710/9f0fd8db-ff70-45e6-b492-5d42594a455c">

Reply:
For ES|QL histogram, do we use the fetched data or a separate request? 

<img width="600" alt="Screenshot 2024-01-23 at 11 07 48" src="https://github.com/elastic/kibana/assets/1415710/78c23b73-6a86-4fe0-99d9-925c4e4f7ade">

Reply:
A separate request, this optimization is done only for the stats mode (stats, keep) where the histogram is not displayed. The histogram is being computed by another query and not the one the user uses in the unified search

Reply:
@jughosta I agree with Peter. This also exists in main. I created an issue https://github.com/elastic/kibana/issues/175419 to solve it correctly.

Reply:
Makes sense, thanks!